test/clustertest/tests/TimingTest.cpp

```diff
@@ -63,7 +63,7 @@ struct TimingTest : tpunit::TestFixture {
                 ASSERT_GREATER_THAN(peekTime, 0);
                 ASSERT_GREATER_THAN(processTime, 0);
             } else {
-                ASSERT_EQUAL(peekTime, 0);
+                ASSERT_GREATER_THAN(peekTime, 0);
```

Comment:
how did this test even work? do we even hit this block of code?

Reply:
Because `peekTime` was always 0 on the follower after escalation cause the command got torn down and re-created after `peek` had already run.